gojs/src/main/js/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>GoJS Demo</title>
+    <style>
```

Comment:
I think it's a little unfortunate that this style is specified in html. How would we move this into the slinky component?

Reply:
Well, the interface of the `ReactDiagram` component requires a global css class name, so to follow the spirit of that you can use any way of loading css globally, which is a broad topic.

That said, maybe you can do something like this. 

```scala
import typings.react.components.div
import typings.react.mod.CSSProperties

div.style(CSSProperties().setWidth("400px").setHeight("400px"))(
    ReactDiagram(
      divClassName = "non-existing",
      initDiagram = diagram,
      nodeDataArray = nodes,
      skipsDiagramUpdate = false
    )
      .linkDataArray(links)
      .onModelChange(_ => ())
)
```

And then see if the styling you set on the wrapper div is inherited by whatever the diagram draws. You can use some slinky syntax for this, but I only remember how to use this and cant check right now

Reply:
Sadly, it doesn't inherit the style from the parent div. We just get an empty 400x400 div containing another div without the appropriate styling which contains the canvas.

Reply:
Aha, too bad. In that case let's leave the demo as is, this is a quite common problem in the javascript world that you need to load some global css things before things work.